frontend/src/fixtures/ucsbSubjectsFixtures.js

```diff
@@ -1,39 +0,0 @@
-const ucsbSubjectsFixtures = {
```

Comment:
Why are we deleting this file?

Reply:
Accidental commit for another PR was reverted making it look like the file is deleted